extensions/control-plane/store/sql/contract-negotiation-store-sql/src/main/java/org/eclipse/edc/connector/store/sql/contractnegotiation/SqlContractNegotiationStoreExtension.java

```diff
@@ -29,10 +29,12 @@
 
 import java.time.Clock;
 
-@Provides({ ContractNegotiationStore.class })
-@Extension(value = "SQL contract negotiation store")
+@Provides({ContractNegotiationStore.class})
```

Comment:
Whitespace seems to be still missing. See https://github.com/eclipse-edc/Connector/pull/2840#discussion_r1182185925